spring-ai-alibaba-core/src/test/java/com/alibaba/cloud/ai/dashscope/audio/DashScopeAudioTranscriptionModelIT.java

```diff
@@ -0,0 +1,113 @@
+package com.alibaba.cloud.ai.dashscope.audio;
+
+import com.alibaba.cloud.ai.autoconfigure.dashscope.DashScopeAutoConfiguration;
+import com.alibaba.cloud.ai.dashscope.api.DashScopeAudioTranscriptionApi;
+import com.alibaba.cloud.ai.dashscope.common.DashScopeException;
+import org.springframework.ai.audio.transcription.AudioTranscriptionPrompt;
+import org.springframework.ai.audio.transcription.AudioTranscriptionResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.UrlResource;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.test.context.TestPropertySource;
+import reactor.core.publisher.Flux;
+
+import java.net.MalformedURLException;
+import java.util.Objects;
+import java.util.concurrent.CountDownLatch;
+
+@TestPropertySource("classpath:application.yml")
+@SpringBootTest(classes = DashScopeAutoConfiguration.class)
+public class DashScopeAudioTranscriptionModelIT {
+
+	private static final Logger logger = LoggerFactory.getLogger(DashScopeAudioTranscriptionModelIT.class);
+
+	private static final String TEST_URL = "https://dashscope.oss-cn-beijing.aliyuncs.com/samples/audio/paraformer/hello_world_female2.wav";
+
+	private static final String TEST_PATH = "/Users/zhiyi/data/pcm/tts_guijie_16k_s16.pcm";
```

Comment:
imo, yhe path here can be better positioned under the relative resources path of the current test class.